assignments/prerequisites/python-programming-advanced.ipynb

```diff
@@ -37,20 +37,23 @@
    ]
```

Comment:
<p>Please make sure you are not submitting any change you dont want. Revert this file.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/340/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>